core/pkg/ingress/annotations/sessionaffinity/main.go

```diff
@@ -31,7 +31,7 @@ const (
 	// If a cookie with this name exists,
 	// its value is used as an index into the list of available backends.
 	annotationAffinityCookieName = "ingress.kubernetes.io/session-cookie-name"
-	defaultAffinityCookieName    = "route"
+	defaultAffinityCookieName    = "INGRESSCOOKIE"
```

Comment:
@bprashanth I've got this, but no reference to sticky / cookie session:

https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html

I've seen in there also that it's common to use something like CamelCase (IngressCookie) as an example.

Headers containing X- are going to be deprecated, as the following RFC:

https://tools.ietf.org/html/rfc6648

So we can keep this as all caps by now (as does Google), and reopen this discussion later?

The RFC section that deals with Requests and Responses Headers is the following: https://tools.ietf.org/html/rfc2616#section-5.3